### PR TITLE
Add validation for candidates

### PR DESCRIPTION
**Pull request by pbochis**
- add validateable behaviour to `invite-candidate-input`

### CODE REVIEW

File: app/elements/candidates/invite-candidate-input.html [hunk 1]
```diff
@@ -71,13 +83,31 @@
...
+					type: String,
+					value: 'self-center'
+				}
+			},
+			selectClass: function(){
+				return this.invalidSelect ? 'invalid self-center' : 'self-center';
+			},
+			validate: function(){
+				var valid = true;
+				//TODO: When https://github.com/PolymerElements/paper-dropdown-menu/pull/57 gets released, revisit this.
```
Comment (lorenzleutgeb) [inline app/elements/candidates/invite-candidate-input.html]: I just left them a comment. Releasing a new version should not be too cumbersome, maybe they are nice enough to release this in the next days.

**Comment by pbochis** [comment]
This should be merged asap. Because of [this](https://github.com/Polymer/polymer/issues/1735) changing all the selects in the app with `paper-dropdown-menu` is critical for IE compatibility.

**Comment by lorenzleutgeb** [comment]
A new version of `paper-dropdown-menu` [was released](https://github.com/PolymerElements/paper-dropdown-menu/commit/63b8200dc68ce297dcf2000a60587f3f68464f31) it should include the fixes you mention in your changes. From my understanding it would make sense to migrate to that version now and merge afterwards.

File: app/elements/candidates/invite-candidate-input.html [hunk 2]
```diff
@@ -1,68 +1,82 @@
...
-		<div id="inputContainer">
-			<paper-input id="address" label="Address" value="{{candidate.Address}}"></paper-input>
+		<div class="horizontal layout">
+			<paper-input
+				class="self-center"
+				id="address"
+				label="Name <email>"
+				value="{{candidate.Address}}"
+				pattern="[a-zA-Z ]*<[a-z0-9!#$%&\'*+\/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&\'*+\/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?>"
+				error-message="Incorrect format"
```
Comment (lorenzleutgeb) [inline app/elements/candidates/invite-candidate-input.html]: _Useless_ message! At least give a hint on how it should look like.

File: app/elements/candidates/invite-candidate-input.html [hunk 3]
```diff
@@ -1,68 +1,82 @@
...
 		}
 	</style>
 	<template>
-		<div id="inputContainer">
-			<paper-input id="address" label="Address" value="{{candidate.Address}}"></paper-input>
+		<div class="horizontal layout">
+			<paper-input
+				class="self-center"
+				id="address"
+				label="Name <email>"
```
Comment (lorenzleutgeb) [inline app/elements/candidates/invite-candidate-input.html]: `Firstname Lastname <person@example.com>`

**Pull request merged**